spec/requests/entry_requests_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
```

Comment:
Please remove this `require`, it is already included in the rspec. You can add the `# frozen_string _literal: true` instead.